lib/create-contentful-api.ts

```diff
@@ -344,12 +343,12 @@ export default function createClientApi(makeRequest: MakeRequest) {
      * .catch(console.error)
      * ```
      */
-    rawRequest: function rawRequest(opts: AxiosRequestConfig & { url: string }) {
+    rawRequest: function rawRequest({ url, ...config }: AxiosRequestConfig & { url: string }) {
       return makeRequest({
         entityType: 'Http',
         action: 'request',
-        params: { url: opts.url, config: opts },
-      }).then((response) => response.data, errorHandler)
```

Comment:
Why did we remove this error handler here?

Reply:
The error handling is http specific and therefore part of the endpoint code: https://github.com/contentful/contentful-management.js/blob/8b3d421da8b1cf56d92ec15e3b9616e5c668a5fb/lib/adapters/REST/endpoints/raw.ts#L61